_episodes/dataflow.md

```diff
@@ -17,3 +17,192 @@ learners should be able to
 __explain how a workflow document describes the input and output of a workflow and the flow of data between tools__
 
 {% include links.md %}
+
+
+
+CWL Workflows are about chaining different steps.
+
+A workflow is the orchestration of the individual steps.
+
+
+
+## Definitions
+
+> A CWL Tool description is a document that enables to run a command line tool, using a CWL runner (i.e. any type of workflow engine that is CWL compatible)
+
+> A CWL Workflow description is a particular chaining of steps (running CWL Tools or CWL Workflows) that enables a their execution with a CWL runner in an orchestrated way.
+
+
+This is what you see in a header of a CWL Tool Description
+
+(link: https://github.com/common-workflow-library/bio-cwl-tools/blob/release/bwa/BWA-Index.cwl)
+
+```
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  DockerRequirement:
+    dockerPull: "quay.io/biocontainers/bwa:0.7.17--ha92aebf_3"
+  InlineJavascriptRequirement: {}
+
+inputs:
+  InputFile:
+    type: File
+    format: edam:format_1929  # FASTA
+    inputBinding:
+      position: 200
+    
+  IndexName:
```

Comment:
Not sure why the IndexName needs to be created for a simple example, couldn't it just be the string itself?